q2cli/handlers.py

```diff
@@ -310,12 +310,25 @@ def get_click_options(self):
 
     def get_value(self, arguments, fallback=None):
         import qiime2
-
+        import qiime2.sdk.util
```

Comment:
it looks like the `sdk.util` subpackage gets loaded when you import `qiime2`, so maybe we can skip this import.